src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/helpers/AddressHelper.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.helpers
+
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Address
+
+fun generateTestAddress(
+  postcode: String = "SE1 1TZ",
+  country: String = "England",
+  county: String = "Greater London",
+  endDate: String = "20 May 2023",
+  startDate: String = "10 May 2021",
+  locality: String = "London Bridge",
+  name: String = "The chocolate factory",
+  number: String = "89",
+  street: String = "Omeara",
+  town: String = "London Town",
+  type: String = "Type?",
```

Comment:
Is this expected? Could we put a realistic value or we're not sure about what that might be atm?

Reply:
Yeah tricky this one, Prison has this field but probation doesn't. 
Investigation currently underway to figure out what to do with this field.

No idea what sort of fake value I could give this. Left awkward on purpose so it's obvious it needs attention.